src/OptFrame/HeuristicFactory.hpp

```diff
@@ -72,8 +72,8 @@
 #include "ComponentBuilder.h"
 
 
-using namespace std;
-using namespace optframe;
+//using namespace std;
+//using namespace optframe; (?????????????????) Don't use namespace declarations in headers
```

Comment:
Very good... I know that... but it's time to acomplish a more "responsible" coding ;)

Reply:
I always have several issues with this inside my own projects. Two good goals for this project should be: 

--making a setup for windows and a makefile for unix for keeping the code with other C libraries, like openssl and std::
--Separating implementation from interface with .cpp and .h files.